api/src/main/resources/marquez/db/migration/V41__alter_jobs_add_job_parent_id.sql

```diff
@@ -0,0 +1,33 @@
+ALTER TABLE jobs ADD COLUMN IF NOT EXISTS parent_job_id uuid CONSTRAINT jobs_parent_fk_jobs REFERENCES jobs (uuid);
+ALTER TABLE runs ADD COLUMN IF NOT EXISTS parent_run_id uuid CONSTRAINT runs_parent_fk_runs REFERENCES runs (uuid);
```

Comment:
_Minor_: We use `*_uuid` internally for IDs within the database (when referencing the primary key). For example, the API returns a `run.id`, but in the database it's really just the `uuid`. This also avoids any confusion with `jobId` or `datasetId`s which is a `namespace` + `name` pairing.

for consistency, I recommend using:

```
parent_job_uuid
parent_run_uuid
```